openhtf/util/measurements.py

```diff
@@ -151,15 +151,15 @@ def WithArgs(self, **kwargs):
     new_meas.docstring = formatter(self.docstring)
     return new_meas
 
-  def __getattr__(self, attr):
+  def __getattr__(self, attr):  # pylint: disable=invalid-name
     """Support our default set of validators as direct attributes."""
     # Don't provide a back door to validators.py private stuff accidentally.
     if attr.startswith('_') or not hasattr(validators, attr):
       raise AttributeError("'%s' object has no attribute '%s'" % (
           type(self).__name__, attr))
 
     # Create a wrapper to invoke the attribute from within validators.
-    def _WithValidator(*args, **kwargs):
+    def _WithValidator(*args, **kwargs):  # pylint: disable=invalid-name
```

Comment:
same here, I think this should be a pylintrc change instead. That can be a separate PR entirely since we have so many lint issues at the moment